hyper/hyper/save_test.go

```diff
@@ -46,7 +49,13 @@ func TestSave(t *testing.T) {
 				defer desktop.Free()
 			}
 
-			err = h.Save(desktop, "/tmp/test.dump")
+			dir, err := ioutil.TempDir("", "dumps")
+			if err != nil {
+				log.Fatal(err)
+			}
+			defer os.RemoveAll(dir)
```

Comment:
This should be inside the `t.Run()`. Also, if you `defer os.RemoveAll(dir)` here will remove the directory before the test